Dockerfile

```diff
@@ -292,7 +292,7 @@ RUN git clone https://github.com/creationix/nvm.git ~/.nvm && \
 ENV ELM_VERSION=0.19.0-bugfix6
 ENV YARN_VERSION=1.22.4
 
-ENV NETLIFY_NODE_VERSION="12.18.0"
+ENV NETLIFY_NODE_VERSION="14.16"
```

Comment:
Should we only pin the major instead?
Otherwise, we might miss some security patches, if the version is stuck to a previous minor version.

For example, if `14.17.0` was released with some non-breaking features, then `14.17.1` was released with some security fixes, we would miss those by pinning to `14.16`.